### DISCUSSION TITLE
Scoop and quarkus-cli: has version names dropped the Final word ?

### DISCUSSION THREAD

**Discussion by yostane** [quarkusio/quarkus#37459] [Q&A] [answered]
Hi,

The scoop installer currentmy installs version 3.2.4.Final of quarkus-cli. It looks like version 3.6 does not have Final in it. 
There is an issue in scoop related to this. https://github.com/ScoopInstaller/Main/issues/5229

Do you know if "Final" has been dropped from version names ?

Thanks,

[a comment by an automated account is not shown]

**Comment by gsmet** [comment, marked as answer]
Hi,

Yes, we dropped the `.Final` suffix for Quarkus 3.3. See https://quarkus.io/blog/quarkus-3-3-0-released/ for more details.

**Reply by yostane** [reply]
Thanks for the quick reply :)
